packages/core/test/pseudo-states.spec.ts

```diff
@@ -626,6 +649,35 @@ describe('pseudo-states', () => {
                 });
             });
 
+            it('should warn when pseudo-class expects params but none was given', () => {
+                const config = {
+                    entry: `/entry.st.css`,
+                    files: {
+                        '/entry.st.css': {
+                            namespace: 'entry',
+                            content: `
+                            .my-class {
+                                -st-states: state1(string);
+                            }
+                            |.my-class:state1 {}
+                            `,
+                        },
+                    },
+                }
+
+                const res = expectWarningsFromTransform(config, [
+                    {
+                        message: 'pseudo-state "state1" defined with unknown type: "string"',
```

Comment:
use the same `stateErrors,.*` to create the expected diagnostic message

Reply:
Used new one.

Reply:
What Ido is saying here is use `stateErrors.UNKNOWN_STATE_TYPE` and don't repeat the error this way when the error is changed you don't need to change the test